src/main/java/org/sonarlint/intellij/ui/SonarLintRulePanel.kt

```diff
@@ -204,28 +203,26 @@ class SonarLintRulePanel(private val project: Project, private val parent: Dispo
             updateHeader(finding, ruleDetails)
             descriptionPanel.removeAll()
             val fileType = RuleLanguages.findFileTypeByRuleLanguage(ruleDetails.language.languageKey)
-            ruleDetails.description.map(
-                { monolithDescription ->
-                    val scrollPane = parseCodeExamples(monolithDescription.htmlContent, fileType)
-                    descriptionPanel.add(scrollPane, BorderLayout.CENTER)
-                },
-                { withSections ->
-                    val htmlHeader = withSections.introductionHtmlContent
-                    if (!htmlHeader.isNullOrBlank()) {
-                        val htmlViewer = RuleHtmlViewer(false)
-                        descriptionPanel.add(htmlViewer, BorderLayout.NORTH)
-                        htmlViewer.updateHtml(htmlHeader)
-                    }
-
-                    val sectionsTabs = JBTabbedPane()
-                    sectionsTabs.font = UIUtil.getLabelFont().deriveFont(Font.BOLD)
-
-                    withSections.tabs.forEachIndexed { index, tabDesc ->
-                        addTab(tabDesc, sectionsTabs, index, fileType)
-                    }
-
-                    descriptionPanel.add(sectionsTabs, BorderLayout.CENTER)
-                })
+            ruleDetails.description.map({ monolithDescription ->
```

Comment:
Is there any important change in this class except re-formatting ? Maybe we don't all have the same style settings

Reply:
One important change on method parameters, but otherwise no, I removed all the indentation modifications